lib/main.dart

```diff
@@ -18,24 +19,82 @@ class Interprep extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp(
       title: "Interprep",
-      home: Scaffold(
-        appBar: AppBar(
-          title: Text("Interprep"),
-          centerTitle: true,
-          backgroundColor: Colors.brown[300],
-          actions: <Widget>[
-            IconButton(
-              icon: Icon(
-                Icons.book,
-                color: Colors.white,
+      home: MainRouter(),
+      debugShowCheckedModeBanner: false,
+    );
+  }
+}
+
+class MainRouter extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Interprep"),
+        centerTitle: true,
+        backgroundColor: Colors.brown[300],
+        actions: <Widget>[
+          FlatButton(
+            textColor: Colors.white,
+            child: Text("About", style: TextStyle(fontWeight: FontWeight.bold)),
+            shape: CircleBorder(side: BorderSide(color: Colors.transparent)),
+            onPressed: () {
+              Navigator.push(context,
+                  MaterialPageRoute(builder: (context) => AboutInterface()));
+            },
+          ),
+          IconButton(
+            icon: Icon(
+              Icons.book,
+              color: Colors.white,
+            ),
+            onPressed: null,
+          ),
+        ],
+      ),
+      body: CardInterface(),
+    );
+  }
+}
+
+List<String> aboutInfo = AboutInfo.aboutInfo;
+
+List<Widget> widgets = aboutInfo
+    .map((str) => Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: (str[0] == '@')
+              ? new Text(str, style: TextStyle(fontWeight: FontWeight.bold))
+              : new Text(str),
+        ))
+    .toList();
```

Comment:
Can you put this inside `AboutInterface`, and `widgets` is too vague of a name

Reply:
fixed